python/ray/train/_internal/logging.yaml

```diff
@@ -0,0 +1,36 @@
+version: 1
+disable_existing_loggers: False
+
+formatters:
+  ray:
+    format: "%(asctime)s\t%(levelname)s %(filename)s:%(lineno)s -- %(message)s"
+  ray_json:
+    class: ray._private.ray_logging.formatters.JSONFormatter
+
+filters:
+  console_filter:
+    (): ray.train._internal.logging.HiddenRecordFilter
+  core_context_filter:
+    (): ray._private.ray_logging.filters.CoreContextFilter
+
+handlers:
+  file:
+    class: ray.train._internal.logging.SessionFileHandler
+    formatter: ray
+    filename: ray-train.log
+  file_json:
+    class: ray.train._internal.logging.SessionFileHandler
+    formatter: ray_json
+    filename: ray-train.log
+    filters: [core_context_filter]
+  console:
+    class: ray._private.log.PlainRayHandler
+    formatter: ray
+    level: INFO
+    filters: [console_filter]
+
+loggers:
+  ray.train:
+    level: DEBUG
```

Comment:
Let's do INFO by default. DEBUG is too spammy.

Reply:
For specifically set the console log handller to be `INFO` level to avoid spammy. This default logger which writes logs to the log file is set to DEBUG level to include more informaiton.